fuel_test/prep_env.sh

```diff
@@ -202,6 +202,33 @@ function wait_for_fm {
 	done
 }
 
+function prepare_centos_repo_and_secret_key {
+    # Create CentOS base repo and import secret key for building supplemental packages
+    local fm_ip="$1"
+    local gpg_secret_key_url="$2"
+
+    ssh -qo StrictHostKeyChecking=no root@$fm_ip \
+    '
+    mkdir /tmp/packages
```

Comment:
use "mkdir -p" just in case this directory exists; and delete this temporary dir in the end.